.github/CODEOWNERS

```diff
@@ -15,8 +15,6 @@
 # limitations under the License.
 #
 
-# This file is only to override the production configurations in standalone server.
-
-registry.plugin.dir=./dolphinscheduler-dist/target/dolphinscheduler-dist-2.0.0-SNAPSHOT/lib/plugin/registry/zookeeper
-registry.plugin.name=zookeeper
-registry.servers=127.0.0.1:2181
+dolphinscheduler/dolphinscheduler-e2e @kezhenxu94
```

Comment:
is it necessary?  '@kezhenxu94'

Reply:
> is it necessary? '@kezhenxu94'

This is an example to list "maintainers" for some specific modules, github will automatically request their review when these modules are modified in pull requests, you and other maintainers can also add the modules you are responsible for. Does it make sense to you?